bot/exts/events/advent_of_code/_cog.py

```diff
@@ -178,21 +180,35 @@ async def join_leaderboard(self, ctx: commands.Context) -> None:
         else:
             await ctx.message.add_reaction(Emojis.envelope)
 
-    @in_month(Month.DECEMBER)
```

Comment:
Why was this removed?

Reply:
Oops, yes. I wanted to test it out quickly before pushing a small change previously and I forgot to put it back.